Competition2017/src/competition/subsystems/shooter_belt/commands/RunBeltSlowlyCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.shooter_belt.commands;
+
+import competition.subsystems.shooter_belt.ShooterBeltSubsystem;
+import competition.subsystems.shooter_wheel.ShooterWheelSubsystem;
+
+public class RunBeltSlowlyCommand extends BaseShooterBeltCommand {
```

Comment:
Can we use the word "tracer" in the name and point out that it only runs when the wheel is at speed?